python/tests/horovod/runner_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import absolute_import, division, print_function
+
+import contextlib
+import logging
+import sys
+
+from six import StringIO
+
+from sparkdl import HorovodRunner
+from ..tests import PythonUnitTestCase
+
+
```

Comment:
These patches would be so much easier had we been testing only against python3. just saying :P

Reply:
cc @ludatabricks